lib/widgets/message_list.dart

```diff
@@ -467,6 +467,7 @@ class _MessageListState extends State<MessageList> with PerAccountStoreAwareStat
       //   will have a `MediaQuery.removePadding` with `removeBottom: true`.
       bottom: false,
 
+      // (center horizontally on wide screens; a no-op in the vertical direction)
```

Comment:
Do you have a screenshot when that happens? I wasn't able to reproduce the effect if I remove `Center` on my devices:

<details>
<summary>Linux</summary>

![image](https://github.com/user-attachments/assets/b4cbc0de-de1c-48da-97b5-0b4173aefe31)


</details>

<details>
<summary>Android</summary>

![1000015248](https://github.com/user-attachments/assets/4e4dc6ec-deba-43b3-9d86-4e36af060c3e)


</details>

Maybe I looked at the wrong place? Oh! I see, I can reproduce that when the compose box is absent in the combined feed, because in that case the message list is the only child in the `Column` on `MessageListPage`, and the message list doesn't fill horizontally like `ComposeBox` does, if without `Center`.

<details>
<summary>Details</summary>

![image](https://github.com/user-attachments/assets/8998c707-7d02-4c25-b3d5-7683943845bc)

</details>

Reply:
If we remove the `Center` here, and keep the one on `MessageListPage`, the combined feed looks fine. Because `Center` specifically deals with sizing the widget within a `Scaffold`, it might be clearer to remove the `Center` here, and comment on the `Center` on `MessageListPage`.